### ISSUE TITLE
Why the NELL dataset has much more features than 5414?

### ISSUE THREAD

**Issue by ranery** [rusty1s/pytorch_geometric#2392]
From now on, we recommend using our discussion forum (https://github.com/rusty1s/pytorch_geometric/discussions) for general questions.

## ❓ Questions & Help

Hi, I use the default dataset loading to load the NELL dataset, but find that the feature dimension is 61278, much larger than the reported 5414, do you have any thought on that issue?

best

**Comment by rusty1s** [comment]
We follow the experimental setup of [GCN](https://arxiv.org/pdf/1609.02907.pdf), where entity node features are enhanced by a unique one-hot representation for every relation in the dataset. The NELL paragraph in the GCN paper gives additional details.

**Comment by ranery** [comment]
Thanks for your timely response!

I tried with the enhanced features but the GCN accuracy was only ~58%, less than the reported 66%, do you have any suggestions for boosting the accuracy to the matched level?

**Comment by rusty1s** [comment]
Have you modified the hyperparameters for NELL, see [here](https://github.com/tkipf/gcn/issues/14#issuecomment-338056182)? It also seems hard to reproduce the official reported results.

**Comment by ranery** [comment]
Thanks for sharing, yes the training is not stable, actually, the first 20 epochs can achieve 60+% accuracy while further training will drop the accuracy a lot.

Meanwhile, I tried other GCN architecture, i.e., GAT, which can easily achieve comparable or better accuracy than the reported in GCN paper. Is that common?
